discovery-provider/src/models/tracks/track.py

```diff
@@ -24,40 +25,52 @@
 class Track(Base, RepresentableMixin):
     __tablename__ = "tracks"
 
-    blockhash = Column(String, ForeignKey("blocks.blockhash"), nullable=True)
-    blocknumber = Column(Integer, ForeignKey("blocks.number"), nullable=True)
-    slot = Column(Integer, nullable=True)
-    txhash = Column(String, default="", nullable=False)
-    track_id = Column(Integer, nullable=False)
-    is_current = Column(Boolean, nullable=False)
+    blockhash = Column(ForeignKey("blocks.blockhash"))  # type: ignore
+    track_id = Column(Integer, primary_key=True, nullable=False)
+    is_current = Column(Boolean, primary_key=True, nullable=False)
     is_delete = Column(Boolean, nullable=False)
-    owner_id = Column(Integer, nullable=False)
-    route_id = Column(String, nullable=False)
-    title = Column(Text, nullable=True)
-    length = Column(Integer, nullable=True)
-    cover_art = Column(String, nullable=True)
-    cover_art_sizes = Column(String, nullable=True)
-    tags = Column(String, nullable=True)
-    genre = Column(String, nullable=True)
-    mood = Column(String, nullable=True)
-    credits_splits = Column(String, nullable=True)
-    remix_of = Column(postgresql.JSONB, nullable=True)
-    create_date = Column(String, nullable=True)
-    release_date = Column(String, nullable=True)
-    file_type = Column(String, nullable=True)
-    description = Column(String, nullable=True)
-    license = Column(String, nullable=True)
-    isrc = Column(String, nullable=True)
-    iswc = Column(String, nullable=True)
-    track_segments = Column(postgresql.JSONB, nullable=False)
-    metadata_multihash = Column(String, nullable=True)
-    download = Column(postgresql.JSONB, nullable=True)
+    owner_id = Column(Integer, nullable=False, index=True)
+    title = Column(Text)
+    length = Column(Integer)
+    cover_art = Column(String)
+    tags = Column(String)
+    genre = Column(String)
+    mood = Column(String)
+    credits_splits = Column(String)
+    create_date = Column(String)
+    release_date = Column(String)
+    file_type = Column(String)
+    metadata_multihash = Column(String)
+    blocknumber = Column(ForeignKey("blocks.number"), index=True)  # type: ignore
+    track_segments = Column(JSONB(), nullable=False)
```

Comment:
interesting that it was `postgresql.JSONB` (no method) and now it's `JSONB()`

Reply:
Something about the render code here probably does it https://github.com/agronholm/sqlacodegen/blob/bac36f6b704bbde75e38ad14146b679e3fdf0278/src/sqlacodegen/generators.py#L226


Probs the JSON stuff has a default empty arg or something like that